webassembly.html

```diff
@@ -71,8 +71,10 @@ <h1 id="title">[DRAFT] WebAssembly Working Group Charter</h1>
 
       <p class="mission">The <strong>mission</strong> of the <a
         href="https://www.w3.org/community/webassembly/">WebAssembly Working
-        Group</a> is to standardize a portable, size- and load-time-efficient
-        format suitable for compilation to the web.</p>
+        Group</a> is to standardize a size- and load-time-efficient
+        format and execution environment,
+        that allow compilation to web with consistent behavior across a
```

Comment:
that allow compilation to web with consistent behavior across a“

`s/to web/to the web`

Also maybe consider just _allowing_ instead of _that allow_:

➡ _allowing compilation to the web_